gtsfm/config.yaml

```diff
@@ -0,0 +1,37 @@
+SceneOptimizer:
+  _target_: gtsfm.scene_optimizer.SceneOptimizer
```

Comment:
do we need all these targets?

Reply:
@ayushbaid we need them to be able to instantiate instantly without manually calling the constructors

Reply:
got it. Thanks!